Demos/04-add-msgraph/README.md

```diff
@@ -1,6 +1,6 @@
 # Integrate Microsoft Graph into the Application
 
-The last step is to incorporate the Microsoft Graph into the application. For this application, you will use the Microsoft Graph Android SDK.
+The last step is to incorporate the Microsoft Graph into the application. For this application, you will use the Microsoft Graph Java SDK.
```

Comment:
This file still calls it Android SDK in other places. Make sure it's updated everywhere

Reply:
Thanks for suggestions. I updated all other places and also made changes for all following review comments. Please kindly review it again. Thanks.